src/iotjs_binding.cpp

```diff
@@ -139,142 +115,126 @@ JObject::~JObject() {
 }
 
 
-static JRawValueType jnull_val = JVal::Null();
-static JObject jnull(&jnull_val);
-JObject& JObject::Null() {
-  return jnull;
+JObject JObject::Null() {
+  return JObject(JVal::Null(), false);
```

Comment:
I think the following could work here:
```c++
JObject& JObject::Null() {
  static JObject null(JVal::Null(), false);
  return null;
}
```

This should do lazy initialization upon first call to the method (and return a reference, just like in previous version, which would enable a cleaner use of this method).